### PR TITLE
Sachit nav stack fix

### PR DESCRIPTION
**Pull request by SachitM**

### CODE REVIEW

File: gui.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+conda deactivate & conda deactivate & python src/state_machine/src/main_gui.py
```
Comment (UmaArunachalam8) [inline gui.sh:1]: why deactivate twice?

Comment (SachitM) [reply gui.sh:1]: Thats for my PC, I have to deactivate twice

File: src/align_navigation/scripts/undocking.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 Path = 'src/align_navigation/scripts/PodLocationServer/'
 import sys
 sys.path.insert(1, Path)
-from PodServer import *
+# from PodServer import *
```
Comment (UmaArunachalam8) [inline src/align_navigation/scripts/undocking.py:25]: can all be commented or uncommented

File: src/align_navigation/scripts/goal_publisher.py [hunk 3]
```diff
@@ -236,37 +311,44 @@ def docking_execution():
             target_distance = math.sqrt(dx*dx + dy*dy)
             diff_angle = goal[2] - pix_bot_theta
             print("Pose_Error (Estimated) (cm,degree): = ", target_distance*100, diff_angle * 180/np.pi)
-            # if dock_error < 0.05:
-            #     print("Dock_Error (Measured) (cm): = ", dock_error*100)
+            if dock_error < 0.05:
+                print("Dock_Error (Measured) (cm): = ", dock_error*100)
```
Comment (UmaArunachalam8) [inline src/align_navigation/scripts/goal_publisher.py:315]: What about angle error? we used to display it before right

File: src/align_navigation/scripts/goal_publisher.py [hunk 4]
```diff
@@ -236,37 +311,44 @@ def docking_execution():
...
             dock_error = 0
             # print(state)
             StateUpdateMsg.TransState = StateOut.State_Verify
             if target_distance > 0.1 or dock_error > 0.20:
```
Comment (UmaArunachalam8) [inline src/align_navigation/scripts/goal_publisher.py:317]: Q: does it not retrace if pose error is high?

File: src/align_navigation/scripts/undocking.py (hunk 2 above)
Comment (SachitM) [reply src/align_navigation/scripts/undocking.py:25]: This was giving an error thats why I commented. You can remove all if not needed

**Review by UmaArunachalam8 — APPROVED** [review]

File: src/align_navigation/scripts/goal_publisher.py (hunk 4 above)
Comment (SachitM) [reply src/align_navigation/scripts/goal_publisher.py:317]: That's there on line 319, this is just a log. Retrace looks at both errors dock and target pose

Comment (SachitM) [reply src/align_navigation/scripts/goal_publisher.py:317]: Actually I am setting dock error as 0, so that only pose error is considered. Let me fix that

**Pull request merged**